src/CommunicationLayer/CommDeviceControl/CommDefines.h

```diff
@@ -4,6 +4,7 @@ namespace CommDefines
 {
     enum Type
     {
-        Udp
+        Udp,
+        Internet
```

Comment:
Internet encapsulates UDP, we shouldn't be keeping both. That doesn't make sense.